WeakAuras/GenericTrigger.lua

```diff
@@ -1342,6 +1342,9 @@ do
         self.count = self.count + 1
         return self.count
       end,
+      SetCount = function(self, count)
```

Comment:
I squashed two very minor thigns into your commit. This new function, and the removal of the
```Private.ParseCron = ParseCron```
line